application.js

```diff
@@ -24,15 +22,14 @@ app.use(express.static(__dirname + '/public'));
 // Note: important that this is added just before your own Routes
 app.use(mbaasExpress.fhmiddleware());
 
-// fhlint-begin: custom-routes
+app.use('/hello', require('./lib/hello.js'));
```

Comment:
do we want to inlude a hello endpoint?
should the template not be concerned with saml stuff only?